local-test-operator.yml

```diff
@@ -54,126 +59,139 @@
         name: install_operator_prereqs
       when: run_prereqs|bool
 
-    - name: "Run operator-courier nest to copy the operator metadata in nested format to the work dir"
-      shell: "operator-courier nest {{ operator_dir }} {{ operator_work_dir }}"
-
-    - name: "Parse operator metadata needed to run the tests"
-      include_role:
-        name: parse_operator_metadata
-
-    - name: "Run linting tests with operator-courier verify on the deployed operator"
+    - name: "Install catalog prerequisites"
       include_role:
-        name: operator_courier_verify
-      when: run_lint|bool
+        name: install_docker
+      when: distro_type == "upstream"
```

Comment:
I think it would make more sense to have these changes in a separate playbook as there are no common roles being used between upstream and OpenShift operators.

Reply:
+1